tests/modules/nf-core/clustalo/align/main.nf

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CLUSTALO_GUIDETREE } from '../../../../../modules/nf-core/clustalo/guidetree/main.nf'
+include { CLUSTALO_ALIGN     } from '../../../../../modules/nf-core/clustalo/align/main.nf'
+
+workflow test_clustalo_align {
+    
+    input = [
+        [ id:'test' ], // meta map
+        file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+    ]
+    
+    CLUSTALO_ALIGN ( input, [[:],[]] )
+}
+
+
+workflow test_clustalo_align_with_tree {
+    
+    input = [
+        [ id:'test' ], // meta map
+        file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+    ]
+
+    ch_tree = CLUSTALO_GUIDETREE ( input ).tree
```

Comment:
```suggestion
    ch_tree = CLUSTALO_GUIDETREE ( input ).tree
```
what is the purpose of this `.tree` at the end ?

Reply:
This is to retrieve the right output channel from that module. The module CLUSTALO_GUIDETREE has 2 channel outputs, tree and versions. 

https://github.com/luisas/modules/blob/9a884757b561688e0b3ff8b55ff7eb4da25eef33/modules/nf-core/clustalo/guidetree/main.nf#L14-L15